src/Tribe/Attendee_Info/Template.php

```diff
@@ -0,0 +1,344 @@
+<?php
+/**
+ * Class Tribe__Tickets__Attendee_Info__Template
+ */
+class Tribe__Tickets__Attendee_Info__Template extends Tribe__Templates {
+
+	/*
+	 * List of themes which we may want to include fixes
+	 */
+	public $themes_with_compatibility_fixes = array(
+		'twentynineteen',
+		'twentyseventeen',
+		'twentysixteen',
+		'twentyfifteen',
+	);
+
+	/**
+	 * Initialize the template class
+	 */
+	public function init() {
+
+		// Spoof the context
+		add_filter( 'the_posts', array( $this, 'setup_context' ), -10 );
+
+		// Set and remove the required body classes
+		add_action( 'wp', array( $this, 'set_body_classes' ) );
+
+		// Choose the wordpress theme template to use
+		add_filter( 'template_include', array( $this, 'set_page_template' ) );
+
+		// Set the content of the page
+		add_action( 'loop_start', array( $this, 'set_page_content' ) );
+
+		// Modify the link for the edit post link
+		add_filter( 'edit_post_link', array( $this, 'set_edit_post_link' ) );
+
+		// Modify the page title
+		add_filter( 'document_title_parts', array( $this, 'modify_page_title' ), 1000 );
+	}
+
+	/**
+	 * Setup the context
+	 *
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	public function setup_context( $posts ) {
+	    global $wp, $wp_query;
+
+	    // Bail if we're not on the attendee info page
+	    if ( empty( $wp->query_vars['attendee-info'] ) ) {
+			return $posts;
+		}
+
+		// Empty posts
+        $posts = null;
+        // Create a fake virtual page
+        $posts[] = $this->spoofed_page();
+
+		// Set it as an archive page so it doesn't give the edit link
+		// nor it loads the comments template
+		$wp_query->is_page     = false;
+        $wp_query->is_singular = false;
+        $wp_query->is_home     = false;
+        $wp_query->is_archive  = true;
+        $wp_query->is_category = false;
+        $wp_query->is_404      = false;
+        $wp_query->found_posts = 1;
+        $wp_query->posts_per_page = 1;
+
```

Comment:
Might be useful to set a custom variable like `tribe_is_past` here: 

- https://github.com/moderntribe/the-events-calendar/blob/1a7b164733efda167d769811006d38274e850641/src/Tribe/Query.php#L171

Reply:
What would be the purpose of having that custom var?

Reply:
To access the property or logic to detect if `is_attendee_info_page` outside of this class by hooking into the `wp_query` instead. So extensibility mainly.

Reply:
Right however is part of the `query_vars` not of the query it self.